### ISSUE TITLE
Chat - Reply to a comment via email doesn’t shows up

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#21430]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
**Issue found when executing PR** https://github.com/Expensify/App/pull/20996
## Action Performed:
1. Find an email with direct message and reply to it
2. Check the chat and look for the email reply

## Expected Result:
The user should be able to reply from the Email

## Actual Result:
Reply to a comment via email doesn’t shows up

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.31.2

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/78819774/a9cf01f7-80fa-405f-9242-18ae182fa3f1

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by s-alves10** [comment]
When I send a reply by email, it goes to the concierge and I wasn't able to receive any events from the backend

**Comment by sakluger** [comment]
I wasn't able to reproduce this. When I respond to a NewDot DM from email, the response shows up in the correct chat.

A few troubleshooting steps I went through:

- When I replied, it sent the reply to `replies+65335546@expensify.com`. That number (`65335546`) matches the chat ID of the DM). The reply email in the example video for this issue is `replies+4391929932843482@expensify.com`. That matches the chat ID in the same video.
- I tested both with a simple text reply and also with a formatted reply with a link, just like the one in this example
    - <img width="865" alt="image" src="https://github.com/Expensify/App/assets/5997400/dd4d62d1-0374-4d7b-87e6-ea0152fe8b83">

@lanitochka17 it there anything else unique about the steps you took here that could help me reproduce?
